components/concept/toolbar/src/main/java/mozilla/components/concept/toolbar/Toolbar.kt

```diff
@@ -162,6 +162,11 @@ interface Toolbar {
      */
     fun addEditActionEnd(action: Action)
 
+    /**
+     * Hide end actions in edit mode.
```

Comment:
LGTM! Could I just get some clarification what "end actions" are?

Reply:
These are actions that are at the end of the awesomebar.  Currently right now it's the voice button.